static/src/javascripts/projects/commercial/modules/ad-free-slot-remove.ts

```diff
@@ -0,0 +1,66 @@
+import once from 'lodash/once';
+import { $$ } from '../../../lib/$$';
+import fastdom from '../../../lib/fastdom-promise';
```

Comment:
I think we can simplify the imports as: 

```suggestion
import { $$ } from 'lib/$$';
import fastdom from 'lib/fastdom-promise';
```

Reply:
I removed all the magic paths from commercial stuff :) #23494 in prep for pulling it out, so commercial-core doesn't need to maintain them too